config/software/make.rb

```diff
@@ -17,8 +17,7 @@
 name "make"
 default_version "4.1"
 
-license "libpng"
-license_file "LICENSE"
+license :project_license
```

Comment:
Why do we need to change this license? Looks like it is a valid license and exactly same with https://github.com/chef/omnibus-software/blob/master/config/software/make.rb ?